P5/Source/Guidelines/en/TC-CriticalApparatus.xml

```diff
@@ -21,18 +21,36 @@ $Id$
     notes at the bottom of each page. TEI’s critical apparatus module allows variation to be encoded
     so that such notes may be generated, but it also models the variation so that, for example,
     interactive editions in which readers can choose which witness readings to display are possible.</p>
-
+  
   <p>Information about variant readings (whether or not represented by a critical apparatus in the
     source text) may be recorded in a series of <term>apparatus entries</term>, each entry
     documenting one <term>variation</term>, or set of readings, in the text. Elements for the
     apparatus entry and readings, and for the documentation of the witnesses whose readings are
     included in the apparatus, are described in section <ptr target="#TCAPLL"/>. Special tags for
     fragmentary witnesses are described in section <ptr target="#TCAPMI"/>. The available methods
     for embedding the apparatus in the rest of the text, or for linking an external apparatus to the
-    base text, are described in section <ptr target="#TCAPLK"/>. Finally, several extra attributes
+    edition text, are described in section <ptr target="#TCAPLK"/>. Finally, several extra attributes
```

Comment:
While saying "base text" here is too partial to one approach, I find "edition text" unclear. Maybe: "to the text being edited"? Though which one?